WeakAuras/AuraEnvironment.lua

```diff
@@ -128,6 +109,28 @@ end
 
 WeakAuras.WA_Utf8Sub = WA_Utf8Sub
 
+-- Wrapping a unit's name in its class colour is very common in custom Auras
+local WA_ClassColorName = function(unit, maxlen)
+  if unit and UnitExists(unit) then
+    local name = WeakAuras.UnitName(unit)
+    if maxlen > 0 then
```

Comment:
this would make a comparison with nil error on any previously made code using this function

Reply:
@mrbuds I updated the PR with the change above. Please let me know if that's still incorrect.